git_pile/git_pile.py

```diff
@@ -1498,7 +1476,8 @@ def _genbranch(root, patchesdir, config, args):
     if not config.check_is_valid():
         return 1
 
-    baseline = args.baseline or get_baseline(patchesdir)
+    pile = Pile(path=patchesdir)
+    baseline = args.baseline or pile.baseline()
```

Comment:
should we "patch" the baseline in the pile object? Otherwise there will be a mismatch in this method that may trigger a bug later

Reply:
By "mismatch", do you mean something later using `pile.baseline()` instead of the variable `baseline`? Yes, that would be a problem, specially if we need to pass the pile object as argument to another function.

We could allow the user to pass an alternative baseline in the constructor of `Pile()`. What do you think?

Reply:
works for me

Reply:
Force-pushed with the proposed change (baseline parameter in the constructor).